docs/recipes.md

```diff
@@ -105,6 +105,18 @@ In your leak reporting code:
 val retainedInstanceCount = AppWatcher.objectWatcher.retainedObjectCount
 ```
 
+## LeakCanary in release builds
+
+We **do not recommend** including LeakCanary in your production build. To avoid accidentally including the `com.squareup.leakcanary:leakcanary-android` dependency in a release build, LeakCanary crashes during initialization if the APK is not debuggable.
```

Comment:
nit: this says both "production" and "release". We should ideally pick one, as changing vocabulary can be confusing. Probably release?

Reply:
Welp, I wanted to make a PR but I pushed to master instead. Oh well. 

https://github.com/square/leakcanary/commit/28d53788055bd219139585cdd89525ade25fd416

🤫 don't tell anyone